locales/fr_FR/LC_MESSAGES/duckduckgo.po

```diff
@@ -20,32 +20,35 @@ msgstr ""
 "X-Generator: Poedit 2.3\n"
 
 msgid "!Bang Search Shortcuts"
-msgstr "Raccourcis !Bang"
+msgstr "Raccourcis de recherche !Bang"
 
+#. input correct way to showcase amounts in French (done)
 msgctxt "showcase_donations"
 msgid "$800,000 in privacy donations!"
-msgstr "800,000 $ de donations privées !"
+msgstr "800 000 $ de dons privés !"
 
 msgctxt "SERP footer content"
 msgid "%s Billion Searches"
 msgstr "%s milliards de recherches"
 
 msgid "%s blocked by safe search."
-msgstr "%s bloqué par la recherche sécurisée."
+msgstr "%s bloqué par le module de recherche en toute sécurité."
```

Comment:
The previous version seemed more accurate and was more succinct. Do you mind reviewing this again to make sure the meaning is correct? If I had to translate from French back to English, this new version would read something like "blocked by the search module, safely", which doesn't really match the original meaning.

Reply:
I'm reverting that change as per your comment. Just to be noted French is a bit more "wordy" than English - less succinct